runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -273,13 +311,8 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 				int itemId = intStack[intStackSize - 1];
 				String search = stringStack[stringStackSize - 1];
 
-				boolean tagSearch = search.startsWith(TAG_SEARCH);
-				if (tagSearch)
-				{
-					search = search.substring(TAG_SEARCH.length()).trim();
-				}
-
-				if (tagManager.findTag(itemId, search))
+				boolean tagSearch = tagManager.isSearchStr(search);
+				if (tagSearch && tagManager.findTag(itemId, search))
```

Comment:
This causes standard bank searches to not search tags. A non `tag:` input should be searching both item name contains and tag name contains.

Reply:
Revert the `isSearchStr` and `getSearchStr`.

Also added tests for the different search scenarios.